docs/developers/listener.md

```diff
@@ -0,0 +1,181 @@
+
+---
+title: Working With Events
+---
+
+Listening to events with Velocity's `@Subscribe` annotation is straightforward. You've already seen one such listener, using the ProxyInitializeEvent in your main class.
+
+### Adding listening methods
+
+To listen to an event, mark the method with `@Subscribe`, like shown. This works similarly to annotation-driven event listening
+in other APIs you may be familiar with; it's the equivalent of Bukkit's/Bungee's @EventHandler and Sponge's @Listener.
+
+```java
+@Subscribe(
+public void onPlayerChat(PlayerChatEvent event) {
+	// do stuff
+}
+```
+
+Note well that the import is `com.velocitypowered.api.event.Subscribe` and *NOT* `com.google.common.eventbus`.
+
+** Orders **
+
+Every listener has a `PostOrder`. When an event is fired, listeners are invoked as ordered by their orders. Listeners using
```

Comment:
Potentially change this to "the order in which listeners are invoked is defined by their `PostOrder`."